tests/data/fields/list_field_test.py

```diff
@@ -91,3 +140,35 @@ def test_as_array_can_handle_multiple_token_indexers(self):
                                                                             [1, 4, 1, 5, 1, 3, 1, 0, 0],
                                                                             [2, 3, 4, 5, 3, 4, 6, 3, 0],
                                                                             [0, 0, 0, 0, 0, 0, 0, 0, 0]]))
+
+    def test_as_array_can_handle_multiple_token_indexers_and_empty_fields(self):
+        # pylint: disable=protected-access
+        self.field1._token_indexers = self.words_and_characters_indexer
```

Comment:
Slight nit on this name - this is actually a dictionary with two indexers.  Could be confusing to someone trying to learn about the library by looking at test cases.